go/base/static/js/src/routing/views.js

```diff
@@ -171,21 +171,49 @@
       this.diagram = options.diagram;
       this.states = this.diagram.states.members.get(this.collectionName);
       this.setElement(this.diagram.$('#' + this.id));
+      this.onDrag = this.onDrag.bind(this);
     },
 
-    repaint: function() { jsPlumb.repaintEverything(); },
+    onDrag: function() {
+      this.repaint();
+    },
 
-    render: function() {
-      this.states.each(function(s) { s.render(); });
+    repaintEl: function($el) {
+      // jsPlumb 1.7.5 doesn't seem to play nice with jquery ui's .sortable(),
```

Comment:
@justinvdm Can we talk about this bit, please? I don't quite get what's happening with `.manage()`. Thanks!